src/main/java/tools/aqua/concolic/Tainting.java

```diff
@@ -7,6 +7,10 @@
 package tools.aqua.concolic;
 
 public class Tainting {
+    
+    // colors for vulnerabilities of securibench-micro
+    static int SQL_INJECTION;
```

Comment:
Actually, these should be fixed constants that hide the concrete taint numbers. Please make them final and initialize them to different integers, e.g., 1 and 2.